keras/backend/common/variables.py

```diff
@@ -175,6 +177,36 @@ def trainable(self):
     def trainable(self, value):
         self._trainable = value
 
+    @property
+    def regularizer(self):
+        return self._regularizer
+
+    @regularizer.setter
+    def regularizer(self, value):
+        from keras.regularizers import Regularizer
+
+        if value is not None and not isinstance(value, Regularizer):
+            raise ValueError(
+                "The object to be assigned must be the subclass of "
```

Comment:
"Invalid value for attribute `regularizer`. Expected an instance of `keras.regularizers.Regularizer`, or `None`. Received: regularizer={regularizer}"

Reply:
Done.
The similar msg has also been applied to `constraint`